### PR TITLE
doc: updated production installation instructions

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

File: dev/INSTALL [hunk 1]
```diff
@@ -11,272 +11,90 @@
...
+      cd /a/www/ietf-datatracker/${releasenumber}
 
-      svn co http://svn.tools.ietf.org/svn/tools/ietfdb/tags/$releasenumber
+ 2. Fetch the release tarball from github 
+    (see https://github.com/ietf-tools/datatracker/releases)::
 
- 2. Don't forget to copy $releasenumber/ietf/settings_local.py from the
-    old release to the new one; otherwise things won't work!
-    ::
+      wget https://github.com/ietf-tools/datatracker/archive/refs/tags/${releasenumber}.tar.gz
```
Comment (jennifer-richards) [inline dev/INSTALL:22]: Need to unpack the tarball (with `--strip-components=1` for consistency with the instructions, I think)

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
I think there's a missing step. The rest looks ok as far as I can tell.

File: dev/INSTALL (hunk 1 above)
Comment (rjsparks) [reply dev/INSTALL:22]: the nonproduction tarballs at least start out in the directory.
```
rjsparks@undex tmp % ls -l release.tar.gz
-rw-r--r--@ 1 rjsparks  staff  49081554 Apr 14 20:47 release.tar.gz
rjsparks@undex tmp % tar tzvf release.tar.gz | head -5
drwxr-xr-x  0 runner docker      0 Apr 14 15:47 ./
drwxr-xr-x  0 runner docker      0 Apr 14 15:46 ./pyzmail/
-rw-r--r--  0 runner docker    863 Apr 14 15:46 ./pyzmail/__init__.py
-rw-r--r--  0 runner docker   5184 Apr 14 15:46 ./pyzmail/utils.py
-rw-r--r--  0 runner docker  32571 Apr 14 15:46 ./pyzmail/parse.py
```

Comment (jennifer-richards) [reply dev/INSTALL:22]: ok, i was looking at 7.46.0, which is likely to be different
```
$ tar tvfz 7.46.0.tar.gz | head
drwxrwxr-x root/root         0 2022-02-23 23:05 datatracker-7.46.0/
-rw-rw-r-- root/root        46 2022-02-23 23:05 datatracker-7.46.0/.bowerrc
drwxrwxr-x root/root         0 2022-02-23 23:05 datatracker-7.46.0/.devcontainer/
-rw-rw-r-- root/root      3491 2022-02-23 23:05 datatracker-7.46.0/.devcontainer/devcontainer.json
-rw-rw-r-- root/root       342 2022-02-23 23:05 datatracker-7.46.0/.devcontainer/docker-compose.extend.yml
-rw-rw-r-- root/root      1208 2022-02-23 23:05 datatracker-7.46.0/.editorconfig
-rw-rw-r-- root/root       421 2022-02-23 23:05 datatracker-7.46.0/.eslintrc.js
drwxrwxr-x root/root         0 2022-02-23 23:05 datatracker-7.46.0/.github/
drwxrwxr-x root/root         0 2022-02-23 23:05 datatracker-7.46.0/.github/workflows/
-rw-rw-r-- root/root      1233 2022-02-23 23:05 datatracker-7.46.0/.github/workflows/dev-db-nightly.yml
```

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**